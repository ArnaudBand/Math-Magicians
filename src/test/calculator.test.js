import renderer from 'react-test-renderer';
import Calculator from '../components/calculator';

it('The calculator rernders correctly', () => {
  const tree = renderer.create(<Calculator />).toJSON();
  expect(tree).toMatchSnapshot();
});
