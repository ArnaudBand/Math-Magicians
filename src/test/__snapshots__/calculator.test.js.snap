// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The calculator rernders correctly 1`] = `
Array [
  <h1
    className="title_home"
  >
    Let's do some maths:
  </h1>,
  <div
    className="calculate"
  >
    <div
      className="container"
    >
      <div
        className="container_calc"
      >
        <div
          className="output"
        >
          <span>
            0
          </span>
          <span />
          <span />
        </div>
        <div
          className="grid-container"
        >
          <button
            className="btn_center"
            id="AC"
            onClick={[Function]}
            type="button"
          >
            AC
          </button>
          <button
            className="btn_center"
            id="+/-"
            onClick={[Function]}
            type="button"
          >
            +/-
          </button>
          <button
            className="btn_center"
            id="%"
            onClick={[Function]}
            type="button"
          >
            %
          </button>
          <button
            className="btn_left"
            id="/"
            onClick={[Function]}
            type="button"
          >
            /
          </button>
          <button
            className="btn_center"
            id="7"
            onClick={[Function]}
            type="button"
          >
            7
          </button>
          <button
            className="btn_center"
            id="8"
            onClick={[Function]}
            type="button"
          >
            8
          </button>
          <button
            className="btn_center"
            id="9"
            onClick={[Function]}
            type="button"
          >
            9
          </button>
          <button
            className="btn_left"
            id="x"
            onClick={[Function]}
            type="button"
          >
            x
          </button>
          <button
            className="btn_center"
            id="4"
            onClick={[Function]}
            type="button"
          >
            4
          </button>
          <button
            className="btn_center"
            id="5"
            onClick={[Function]}
            type="button"
          >
            5
          </button>
          <button
            className="btn_center"
            id="6"
            onClick={[Function]}
            type="button"
          >
            6
          </button>
          <button
            className="btn_left"
            id="-"
            onClick={[Function]}
            type="button"
          >
            -
          </button>
          <button
            className="btn_center"
            id="1"
            onClick={[Function]}
            type="button"
          >
            1
          </button>
          <button
            className="btn_center"
            id="2"
            onClick={[Function]}
            type="button"
          >
            2
          </button>
          <button
            className="btn_center"
            id="3"
            onClick={[Function]}
            type="button"
          >
            3
          </button>
          <button
            className="btn_left"
            id="+"
            onClick={[Function]}
            type="button"
          >
            +
          </button>
          <button
            className="btn_center doublespan"
            id="0"
            onClick={[Function]}
            type="button"
          >
            0
          </button>
          <button
            className="btn_center"
            id="."
            onClick={[Function]}
            type="button"
          >
            .
          </button>
          <button
            className="btn_left"
            id="="
            onClick={[Function]}
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>,
]
`;
